src/abstractdataframe/selection.jl

```diff
@@ -150,19 +150,29 @@ const TRANSFORMATION_COMMON_RULES =
     transformation and single column selection operations must be unique, so e.g.
     `select!(df, :a, :a => :a)` or `select!(df, :a, :a => ByRow(sin) => :a)` are not allowed.
 
-    As a general rule if `copycols=true` columns are copied and when
-    `copycols=false` columns are reused if possible. Note, however, that
-    including the same column several times in the data frame via renaming or
-    transformations that return the same object without copying may create
-    column aliases even if `copycols=true`. An example of such a situation is
-    `select!(df, :a, :a => :b, :a => identity => :c)`.
-    As a special case in `transform` and `transform!` column renaming always
-    copies columns to avoid storing aliased columns in the target data frame.
+    In general columns returned by transformations are stored in the target data
+    frame without copying. An exception to this rule is when columns from
+    the source data frame are reused in the target data frame (via expressions
+    like: `:x1`, `[:x1, :x2]`, `:x1 => :x2`, `:x1 => identity => :x2`, or
+    `:x1 => (x -> @view x[repeat(1, length(x))])`; note that in the last case
+    the source column is reused as its view is returned). In such cases the
```

Comment:
```suggestion
    the source data frame are reused in the target data frame.
    This can happen via expressions like: `:x1`, `[:x1, :x2]`, `:x1 => :x2`,
    `:x1 => identity => :x2`, or `:x1 => (x -> @view x[inds])`
    (note that in the last case the source column is reused
    indirectly via a view). In such cases the
```